src/bootstrap/src/core/build_steps/test.rs

```diff
@@ -1691,6 +1605,9 @@ NOTE: if you're sure you want to do this, please open an issue as to why. In the
                 host: target,
             });
         }
+        if suite == "run-make" {
```

Comment:
```suggestion

        if suite == "run-make" {
```

nit: for consistency with other conditionals in this function.

Reply:
I'll apply this formatting diff in the follow-up

Reply:
It's already in a rollup, I can fix that in some follow-up PR.